### DISCUSSION TITLE
How can I change the MIDI CC mapping?

### DISCUSSION THREAD

**Discussion by KeenVox** [marcel-licence/esp32_fm_synth#6] [Q&A]
Hi,

I build the synth and it works great, but how do I define the cc messages from my controller for the eps32 functions?

Greets 
Keen

**Comment by marcel-licence** [comment]
Hi,

good to hear. For easier mapping you could use the debug output: https://youtu.be/ZNxGCB-d68g?t=979
Hope this snipped helps.

Best regards,
Marcel
